### PR TITLE
Contact me page

### PR DESCRIPTION
**Pull request by gokerlek**
contact me page finished

### CODE REVIEW

File: src/components/image-wrapper/image-wrapper.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import './image-wrapper.css';
+export const ImageWrapper = ({ children }) => {
```
Comment (onerzafer) [inline src/components/image-wrapper/image-wrapper.js:2]: -As I can see the Image wrapper is used for icons an links. In general this is a good idea. But semantically the name is not suitable. How about a component name as IconWithLink?

Another optimizations would be not repeating the usage. To do so this component can be changed to this

```javascript
const IconWithLink = ({icon, link}) => {
const IconComponent = icon;
return <a href={link} className="icon-wapper"><IconComponent /></a>
}

```

Therefore the usage would be like

```javascript
<IconWithLink link="https://somelink.com" icon={FaIntagram} />
```
What do you think?

File: src/pages/contact.js [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
 export const ContactPage = () => {
     return <div>
-        Cotact me
-        </div>
+        <h3>Contact me </h3>
+        <p>Hello,</p>
+        <p>I am a web developer</p>
+        <p>If you want to do a project with me</p>
+        <p>Let's keep in touch</p>
+        <p>+90 542 323 92 48</p>
```
Comment (onerzafer) [inline src/pages/contact.js:13]: No phone number online😉 and no email online

If someone would like to reach you they can use the platforms below

**Review by onerzafer — CHANGES_REQUESTED** [review]

File: src/components/icon-component/icon-with-link.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.icon-wapper {
+    width: 1em;
+    height: 1em;
+    overflow: hidden; 
+    margin: 0.5em 1em;
+    display: inline-block;
+    color: #009FB7
+  }
+    .icon-wapper img {
```
Comment (onerzafer) [inline src/components/icon-component/icon-with-link.css:9]: Are the icons img?

Comment (gokerlek) [reply src/components/icon-component/icon-with-link.css:9]: :D

**Review by onerzafer — APPROVED** [review]

**Pull request merged**